frontend/app/templates/components/badge-design.hbs

```diff
@@ -8,14 +8,16 @@
     </div>
     <div class="column features-list">
       <div class="border-left"></div>
+      <div class="align">
```

Comment:
Please intend the lines inside this div. Better move the align class to the child div.